README.md

```diff
@@ -1 +1,20 @@
 👤🎁🔄
+
+## About
+
+A static website that creates gift swop assignments and allows blacklisting of certain pairings from a list of user-provided names.
+The site generates a link to give to each person that states who they will be giving to.
+
+## Frequently Asked Questions
+### Why the British spelling `swop`?
+The term "swop" is a playful variation of "swap" commonly used in British English. We chose this spelling to add a touch of charm and uniqueness to the project, reflecting its lighthearted purpose of gift exchanges. Plus, it helps differentiate our platform while celebrating diverse language conventions!
+
+## Contributing
+
+We welcome contributions from everyone! If you’d like to report a bug, suggest an enhancement, or submit a pull request, please follow these steps: first, ensure your changes align with the project's goals; next, clearly describe your contributions in your pull request; and finally, adhere to our coding style for consistency. Your input is invaluable, so feel free to reach out with any questions or ideas. Together, we can make this project even better!
+
+When creating an issue, please be descriptive! The more information you give us, the better we can understand the problem.
+
+Also, when submitting a pull request, remember to do a couple of things.
+* We use prettier in this project. Please run prettier before submitting a PR to ensure consistent code format
```

Comment:
Again we haven't defined any prettier/formatter yet.

Reply:
Wasnt entirely sure about prettier bc i did see a .prettierrc and .prettier ignore, but I'll go ahead and remove it